pkg/postgres/configuration.go

```diff
@@ -693,6 +702,14 @@ func CreatePostgresqlConfiguration(info ConfigurationInfo) *PgConfiguration {
 		configuration.OverwriteConfig("temp_tablespaces", strings.Join(info.TemporaryTablespaces, ","))
 	}
 
+	if len(info.ImageVolumeExtensions) > 0 {
+		// Set all ExtensionControlPaths
+		setExtensionControlPath(info, configuration)
+
+		// Set all DynamicLibraryPaths
+		setDynamicLibraryPath(info, configuration)
```

Comment:
Is it guaranteed that we'll always have both `extensionControlPath` and `dynamicLibraryPath` if we are inside the block?

Reply:
@hossain-rayhan I think so, given that we want to adhere to the PGXS standard declared in https://www.postgresql.org/docs/devel/extend-pgxs.html, for which every extension (even the most basic ones) will have extension control and SQL files in `/share` and the shared library in `/lib`. So we'll have to set both those GUCs to allow PG to find the extension's files. Although I'm wondering how the build infra would behave in case of pure-SQL extensions. I'll try to test/investigate this specific use case, but I think the worst that could happen is that we'd end up setting a useless path inside `dynamic_library_path`.
Were you thinking of a specific scenario, or was it just a generic question?